viadot/sources/redshift_spectrum.py

```diff
@@ -79,23 +79,26 @@ def rm(
         self,
         database: str,
         table: str,
+        file_rm: bool = True,
```

Comment:
how about `remove_files`?

Reply:
BTW, the file is now called `redshift_spectrum`, while the source is `AWSRedshiftSpectrum`. Pls rename one of them for consistency.

Reply:
> how about `remove_files`?

Updated as per [🎨 updated docstring and arg name for rm() method](https://github.com/dyvenia/viadot/pull/579/commits/28f1d840249a5a3c515cddc140f1bb6251d4a4e5)


> BTW, the file is now called `redshift_spectrum`, while the source is `AWSRedshiftSpectrum`. Pls rename one of them for consistency.

Updated as per [🎨 modified class name to RedshiftSpectrum](https://github.com/dyvenia/viadot/pull/579/commits/f422d78a44f85ebf45db7bca93c4583e33216842)